### ISSUE TITLE
Provide images for building and testing typical Eclipse RCP apps

### ISSUE THREAD

**Issue by mickaelistria** [eclipse-cbi/dockerfiles#3]
Images like ubuntu-gtk3-metacity:18.10-gtk3.24 seem tailored for building and testing UI apps.
Most of the graphical apps in Eclipse ecosystem at the moment are RCP apps or Eclipse plugins, and most (if not all do require Java and Tycho -which are part of the CBI recommended stack-). It would be convenient to have java and Maven installed in those images so they could more easily reused by many projects.

**Comment by mbarbero** [comment]
Agreed that we may want images targeted for typical Eclipse RCP apps. I'll work on that.
